portfolio/src/main/webapp/script.js

```diff
@@ -48,12 +48,20 @@ function newCommentBox(comment){
 }
 
 /**
- * Adds a greeting to the page using Promises
+ * Adds comments to the page using Promises
  */
 function loadComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const commentContainer = document.getElementById('comment-container');
+  const parameters = location.search;
+  // Clear previous children
+  while(commentContainer.lastElementChild) {
```

Comment:
I'm not sure what the best way to clear the comment container dom is. I found this resource https://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript which lists a bunch of options and chose looping + deleting.

Reply:
Yea I tried searching this earlier. I don't mind this. I wonder why there isn't a simple method for this.